openlibrary/macros/databarHistory.html

```diff
@@ -8,7 +8,7 @@
 <div id="editTools" class="edit">
     $if not page.is_fake_record():
         <div id="editButton">
-            <a class="linkButton" href="$edit_url" title="Edit this template" accesskey="e">$_("Edit")</a>
+            <a class="linkButton" href="$edit_url" title="$_('Edit this template')" accesskey="e">$_("Edit")</a>
```

Comment:
`accesskey` ie keyboard shortcut mnemonic needs to be internationalizable to match the button label, but I'm not sure what the best practices are for naming the id.